### PR TITLE
Fixes borers and antag spawning.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
- Fixed bughunt for real.
  - Bughunt now tries to spawn four mercenaries and two borers.
  - Added some more intro text to Renegade.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!bugfix

**Comment by NonQueueingMatt** [comment]
!review

File: code/game/antagonist/antagonist_add.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	if(faction_verb && player.current)
 		player.current.verbs |= faction_verb
 
-	player.current?.client.verbs += /client/proc/aooc
+	player.current?.verbs += /client/proc/aooc
```
Comment (MarinaGryphon) [inline code/game/antagonist/antagonist_add.dm]: Pretty sure player.current is not a client, so I don't know why this change is here. I would do `player.current?.client?.verbs`.

Comment (skull132) [reply code/game/antagonist/antagonist_add.dm]: No, it is. This code runtimes atm because it tries to access "client.client".

**Comment by MarinaGryphon** [comment]
oops too late
